Source/Extensions/Blazorise.FluentValidation/FluentValidationHandler.cs

```diff
@@ -0,0 +1,117 @@
+                var context = CreateContext( validation );
+                var result = validator.Validate( context );
+
+                NotifyValidationChanged( result, validation );
+            }
+            catch ( Exception exc )
+            {
+                HandleException( exc, validation );
+            }
+        }
+
+        public async Task ValidateAsync( IValidation validation, object newValidationValue, CancellationToken cancellationToken = default )
+        {
+            try
+            {
+                var validator = CreateValidator( validation );
+
+                validation.NotifyValidationStarted();
+                var context = CreateContext( validation );
+                var result = await validator.ValidateAsync( context );
+
+                NotifyValidationChanged( result, validation );
+            }
+            catch ( Exception exc )
+            {
+                HandleException( exc, validation );
+            }
+        }
+
+        private void HandleException( Exception exception, IValidation validation )
+        {
+            var msg = $"An unhandled exception occurred when validating field name: '{validation.FieldIdentifier.FieldName}'";
+
+            if ( validation.EditContext.Model != validation.FieldIdentifier.Model )
+            {
+                msg += $" of a child object of type: {validation.FieldIdentifier.Model.GetType()}";
+            }
+
+            msg += $" of <EditForm> model type: '{validation.EditContext.Model.GetType()}'";
+
+            throw new InvalidOperationException( msg, exception );
+        }
+
+        private IValidator CreateValidator( IValidation validation )
+        {
+            var model = validation.FieldIdentifier.Model;
+
+            return TryGetValidatorForObjectType( model.GetType() );
+        }
+
+        private ValidationContext<object> CreateContext( IValidation validation )
+        {
+            var selector = new MemberNameValidatorSelector( new[] { validation.FieldIdentifier.FieldName } );
+
+            return new ValidationContext<object>( validation.FieldIdentifier.Model, new PropertyChain(), selector );
+        }
+
+        private void NotifyValidationChanged( ValidationResult result, IValidation validation )
+        {
+            if ( result.IsValid )
+                validation.NotifyValidationStatusChanged( ValidationStatus.Success );
+            else
+            {
+                var messages = result.Errors.Select( e => e.ErrorMessage );
+                validation.NotifyValidationStatusChanged( ValidationStatus.Error, messages );
+            }
+        }
+
+        private IValidator TryGetValidatorForObjectType( Type modelType )
+        {
+            var validatorType = typeof( IValidator<> );
+            var formValidatorType = validatorType.MakeGenericType( modelType );
+            return serviceProvider.GetRequiredService( formValidatorType ) as IValidator;
```

Comment:
Nowadays everyone uses the netcore DI I guess... 
But maybe there could be a "Reflection" based mode? Where the user would give some indication of the Validators assembly, and the FluentValidationHandler would figure it out?

Reply:
That's done during the setup in `Startup.cs`, eg. `services.AddValidatorsFromAssembly( typeof( App ).Assembly );";
`

Reply:
No, what that does is to register it on DI (with the help of reflection it seems).
I am talking about not using DI at all to find the IValidator? 

Like, let's imagine the user for some reason has two `Person` IValidators. He could do something like:
```
<Validations Validator="EmployeeValidator"/>
<Validations Validator="VisitorValidator"/>
```
Just an idea... not sure how to elegantly implement it.

Reply:
Hmm, yeah, you're right; having more validators for a model is a valid scenario. But considering that we're close to 1.1, I don't think we will have enough time. I will try something.

Reply:
I will do it in 1.2